### ISSUE TITLE
page load speed

### ISSUE THREAD

**Issue by akhmerov** [jupyter-widgets/jupyter-sphinx#62]
Right now google page speed [gives](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fjupyter-sphinx.readthedocs.io) us a score of 3/100.

**Comment by hugokerstens** [comment]
We could load the thebelab javascript library only after clicking on the activate button.

**Comment by akhmerov** [comment]
Good idea. The widgets js might require separate handling, but thebelab would already be an improvement.
